constants/__init__.py

```diff
@@ -161,6 +162,27 @@
             ),
         ],
     ),
+    (
+        3_601_190,
```

Comment:
Can we make a constant for this and use across the 3 places it is used now?

1) for the epsilon experiment, 1 for the competition start, 1 for the sample unpacking block.